game/configuration.cc

```diff
@@ -130,6 +132,13 @@ std::string const ConfigItem::getValue() const {
 		else std::clog << "audio/warning: Currently selected audio backend is unavailable on this system, will default to Auto." << std::endl;
 		return "Auto";
 	}
+	if (this->getName() == "game/language") {
+		int autoLanguageType = 1337;
+		static int val = std::get<int>(m_value);
```

Comment:
why static? seem awkward here....

Reply:
Removed the static and the check on the next line as it didn't provide any value.

Reply:
ok